worker/jobTypes/S3Publish.js

```diff
@@ -20,7 +20,7 @@ class S3PublishClass {
     // check if need to build next-gen
     if (this.GitHubJob.buildNextGen()) {
       if (this.GitHubJob.currentJob.payload.pathPrefix) {
-        stageCommands[stageCommands.length - 1] = `make next-gen-stage ${this.GitHubJob.currentJob.payload.mutPrefix}`;
+        stageCommands[stageCommands.length - 1] = `make next-gen-stage MUT_PREFIX=${this.GitHubJob.currentJob.payload.mutPrefix}`;
```

Comment:
It's best not to rely on shell expression parsing when constructing commands to be run; i.e. better is `["make", "next-gen-stage", "MUT_PREFIX=${...}"]`, rather than relying on the shell to parse things.

This need not block merge.

Reply:
noted! This is historically how we've done commands. I can change this on the next release